editor/create_dialog.cpp

```diff
@@ -208,7 +208,7 @@ void CreateDialog::_update_search() {
 
 	// Select the best result.
 	if (empty_search) {
-		select_type(base_type);
+		select_type(base_type, false);
```

Comment:
Won't that break things when `base_type` is not `Node`? We still want the base type to be in focus.

The problem here is that Node isn't properly in focus, that's likely a themeing or math issue.

Reply:
> Won't that break things when `base_type` is not `Node`?

The resource create dialog suffers from this problem too.

> We still want the base type to be in focus.

IMHO the bool controls whether to center the item, not whether to focus on the item.